eng/Directory.Build.Data.targets

```diff
@@ -42,6 +42,11 @@
     </PackageReference>
   </ItemGroup>
 
+  <!-- Add Package Icon to DataPlane Packages -->
```

Comment:
While the changes match the docs https://docs.microsoft.com/en-us/nuget/reference/errors-and-warnings/nu5048 I worry a little about this. It would be good to test to ensure this correctly shows up on nuget.org when we release. Can we do a release of the Azure.Template package with this change before we merge in this PR? If not I think we should wait until after GA to take this change.

Reply:
Ok, I create a release of Azure Template